api/dataStores.yaml

```diff
@@ -84,6 +86,17 @@ components:
           type: string
         token:
           type: string
+    AzureAppInsights:
+      type: object
+      properties:
+        useAzureActiveDirectoryAuth:
+          type: boolean
+        accessToken:
+          type: string
+        connectionType:
```

Comment:
I think this connection type is something we can move to be part of the base data store object, I don't want to do it now but it will be very useful in the future